### PR TITLE
chore(make): add stop-openapi-demo target

### PR DESCRIPTION
**Pull request by jceb**
Signed-off-by: Jan Christoph Ebersbach <jan-christoph.eberbach@41ppl.com>

**Title:**
Add stop-openapi-demo target

**Description:**
Currently, there's only a target to start the demo but none to shut it down. Since stopping the demo by hand involves multiple commands executed in multiple directories, this new target should reduce the effort to stop the demo.

**Summary:**
Extended the Makefile with a stop-openapi-demo target.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -100,6 +100,12 @@ run-openapi-demo: generate-test-keys generate-openapi-demo-specs
 	@DEMO_COMPOSE_PATH=test/bdd/fixtures/demo/openapi SIDETREE_COMPOSE_PATH=test/bdd/fixtures/sidetree-mock AGENT_REST_COMPOSE_PATH=test/bdd/fixtures/agent-rest  \
         scripts/run-openapi-demo.sh
 
+.PHONY: stop-openapi-demo
+stop-openapi-demo: generate-test-keys generate-openapi-demo-specs
```
Comment (llorllale) [inline Makefile:104]: don't need `generate-test-keys` nor `generate-openapi-demo-specs`

File: Makefile [hunk 2]
```diff
@@ -100,6 +100,12 @@ run-openapi-demo: generate-test-keys generate-openapi-demo-specs
...
+	@echo "Starting demo agent rest containers ..."
```
Comment (llorllale) [inline Makefile:105]: "Stopping"

**Review by llorllale — CHANGES_REQUESTED** [review]
minor changes requested

File: Makefile (hunk 1 above)
Comment (jceb) [reply Makefile]: Whoops, that sloppy.

File: Makefile (hunk 2 above)
Comment (jceb) [reply Makefile]: Whoops, that sloppy.

**Review by llorllale — APPROVED** [review]

**Comment by llorllale** [comment]
@jceb thanks!

**Pull request merged**